app/components/layer-settings/style.scss

```diff
@@ -4,6 +4,7 @@
     color: #fff;
     position: relative;
     height: 42px;
+    z-index: 10;
```

Comment:
it is needed because if the user makes the layer to small the settings would be hidden behind the layer below making that layer unusable